c2corg_ui/externs/appx.js

```diff
@@ -275,6 +275,18 @@ appx.Image;
 
 /**
  * @typedef {{
+ *   locales: Array.<Object>,
```

Comment:
Perhaps. I think I have copied-pasted ``appx.Image``:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/externs/appx.js#L268
I will make the change in  ``appx.Image`` as well.